importers.md

```diff
@@ -32,69 +32,69 @@ O **Importador CSV** permite que usuários importem itens para uma coleção dir
+After configuring the importer and selecting the destination collection, the *.csv* file is uploaded and the user has the chance to map the columns present in the *.csv* to the metadata present in the destination collection. 
 
-> (Veja [como criar metadados automaticamente](#criar-metadados-automaticamente) na seção abaixo para aprender como dizer ao Tainacan o tipo e os atributos de metadados a serem criados.)
+If the metadata was not previously created in the collection, the user can create and map the metadata in this same screen, or choose the option `Create Metadata` in Mapper. If this option is selected, Tainacan will automatically create the metadata when the importer runs.
 
-#### Colunas Especiais
+> (See [how to automatically create metadata](#criar-metadados-automaticamente) in the section below to learn how to tell Tainacan the type and attributes of metadata to create).
 
-Cada coluna do arquivo *.csv* deve ser mapeado para um metadado na coleção de destino. Entretanto, existem colunas especiais que podem ser usadas para configurar outros aspectos do item. Por exemplo, o status do item pode ser configurado como público, rascunho ou privado para editores.
+#### Special Columns
 
-As colunas especiais que podem ser usadas são:
+Each column of the *.csv* file must be mapped to a metadata in the target collection. However, there are special columns that can be used to configure other aspects of the item. For example, the item status can be set to public, draft, or private for editors.
 
-* `special_item_status` - Informa o status do Item. Os valores possíveis são:
-  * `draft`: rascunho
-  * `private`: privado
-  * `publish`: público
-* `special_item_id` - Informa o **ID do item** no banco de dados do Tainacan. Essa função é útil para re-importar itens e permitir que o usuário decida atualizar os itens * existente ou ignora-los e adicionar novos itens.
-* `special_document` - permite que o usuário informe o documento do item. Consulte [Importando arquivos e anexos](#importar-arquivos-e-anexos).
-* `special_attachments` - permite que o usuário informe os anexos. Consulte [Importando arquivos e anexos](#importar-arquivos-e-anexos).
-* `special_comment_status` - permite que o usuário informe se os items podem receber ou não comentários, as opções são: *"open"* ou *"closed"* (padrão).
+The special columns that can be used are:
 
+* `special_item_status` - Informs about the status of the item. Possible values are:
+  * `draft`
+  * `private`
+  * `publish`
+* `special_item_id` - The **item ID** in the Tainacan database. This function is useful for re-importing items and allowing you to decide whether to update existing items or ignore them and add new items.
+* `special_document` - allows you to enter the item document. See [importing files and attachments](#importar-arquivos-e-anexos).
```

Comment:
Here, as you renamed the section a few lines bellow (79) you can refer its slug version, in this case `#import-files-and-attachments`

Reply:
Oh nevermind, you did on the other commit :)